lobby.properties

```diff
@@ -0,0 +1,5 @@
+## uncomment the following lines to have local game runners connect
```

Comment:
Since #649 blocks the lobby either way, I cannot confirm. I looked through the code, but I cannot find a String or enum mentioning a lobby.properties file...
It seems that this file is not used at all...
TripleA uses the online lobby_server.yaml to connect to a lobby.
Is a local lobby even needed for users (developers could just change the URL Constant) - when playing a local game we could just host one using the client...
Can't we just remove the file?